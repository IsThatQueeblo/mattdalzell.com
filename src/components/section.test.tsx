import React from 'react'
import renderer from 'react-test-renderer'

import Section from './section'

describe('Section', () => {
  it('should match basic snapshot', () => {
    const subsections = [
      { content: 'some stuff', heading: 'thing1' },
      { content: 'other stuff', heading: 'thing2' },
    ]
    const component = renderer.create(
      <Section title="Test Section" subsections={subsections} />
    )

    expect(component).toMatchSnapshot()
  })
})
