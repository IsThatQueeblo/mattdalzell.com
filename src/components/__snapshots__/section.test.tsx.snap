// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section should match basic snapshot 1`] = `
<section>
  <h2>
    Test Section
  </h2>
  <dl>
    <dt>
      thing1
    </dt>
    <dd>
      some stuff
    </dd>
    <dt>
      thing2
    </dt>
    <dd>
      other stuff
    </dd>
  </dl>
</section>
`;
